charts/pip-frontend/values.stg.template.yaml

```diff
@@ -2,3 +2,5 @@ nodejs:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+environment:
```

Comment:
this needs to be indented to be within the java app

Reply:
done